app/src/main/java/com/github/onlynotesswent/ui/common/Popups.kt

```diff
@@ -183,62 +186,110 @@ fun FileSystemPopup(
-          Column(modifier = Modifier.fillMaxSize()) {
-            Box(
-                modifier =
+            Column(modifier = Modifier.fillMaxSize()) {
+                Box(
+                    modifier =
                     Modifier.fillMaxWidth()
                         .background(MaterialTheme.colorScheme.primary)
                         .padding(vertical = 12.dp, horizontal = 16.dp)) {
-                  Row(
-                      modifier = Modifier.fillMaxWidth(),
-                      verticalAlignment = Alignment.CenterVertically) {
+                    Row(
+                        modifier = Modifier.fillMaxWidth(),
+                        verticalAlignment = Alignment.CenterVertically) {
                         IconButton(
                             modifier = Modifier.testTag("goBackFileSystemPopup"),
                             onClick = {
-                              if (selectedFolder != null) {
-                                if (selectedFolder!!.parentFolderId != null) {
-                                  folderViewModel.getFolderByIdNoStateUpdate(
-                                      selectedFolder!!.parentFolderId!!,
-                                      onSuccess = { parentFolder -> selectedFolder = parentFolder })
-                                } else {
-                                  selectedFolder = null
+                                if (selectedFolder != null) {
+                                    if (selectedFolder!!.parentFolderId != null) {
+                                        folderViewModel.getFolderByIdNoStateUpdate(
+                                            selectedFolder!!.parentFolderId!!,
+                                            onSuccess = { parentFolder -> selectedFolder = parentFolder })
+                                    } else {
+                                        selectedFolder = null
+                                    }
                                 }
-                              }
                             }) {
-                              Icon(
-                                  imageVector = Icons.AutoMirrored.Filled.ArrowBack,
-                                  contentDescription = "Back")
-                            }
+                            Icon(
+                                imageVector = Icons.AutoMirrored.Filled.ArrowBack,
+                                contentDescription = "Back")
+                        }
                         Spacer(modifier = Modifier.weight(1f))
                         Text(
                             text =
-                                if (selectedFolder == null)
-                                    stringResource(R.string.file_system_folders_in_root)
-                                else "Folders in: ${selectedFolder!!.name}",
+                            if (selectedFolder == null)
+                                stringResource(R.string.file_system_folders_in_root)
```

Comment:
I would change the `R.string.file_system_folders_in_root` string to just `Root` just like how the subfolders only display the name

Reply:
Alright changed the value of the string resource